.eslintrc

```
@@ -90,7 +91,11 @@
         "template-curly-spacing": "error",
         "yoda": "error",
 
-        "lwc-internal/no-invalid-todo": "error"
+        "lwc-internal/no-invalid-todo": "error",
+        "import/order": [
+            "error", { "groups": [[
+                "builtin", "external", "internal"
+            ]] }]
```

Nitpicking on the formatting.

```suggestion
        "import/order": [
            "error", 
            { "groups": [["builtin", "external", "internal"]] }
        ]
```